tests/test_io.py

```diff
@@ -178,7 +181,7 @@ def test_read_fm_map_06(self):
         with pytest.raises(Exception) as cm:
             datac = read_fm_map(filename, varname)
         assert str(cm.value) == 'Expected one variable for "water level", but obtained 0.'
-        
+
```

Comment:
two lines spaced between test functions/classes, and one line space between test functions in the same class.